idrstream/segment.py

```diff
@@ -69,17 +69,19 @@ def get_object_locations(
             dataframe with object center coords
         """
         objects_data = []
-
-        cellpose_model = models.Cellpose(
+        
+        cellpose_model = models.CellposeModel(
             gpu=self.use_GPU, model_type=self.model_specs["model_type"]
         )
-        masks, flows, styles, diams = cellpose_model.eval(
+        
+        masks, flows, *_ = cellpose_model.eval(
```

Comment:
```suggestion
        masks, flows, _ = cellpose_model.eval(
```
I am not familiar with `*_` for unused variables. Conventionally `_` is used for unused variables unless this is a new update I am not aware off. 

For what I know `*` is used to unpack data structures like: `tuples` and `lists`.

Feel free to accept or reject this suggestion!